### DISCUSSION TITLE
v2.3.1

### DISCUSSION THREAD

**Discussion by Zenda-Cross** [vega-org/vega-app#78] [Announcements]
- New Provider AutoEmbed drama, AutoEmbed anime and dooflix
- Searching improved
- Hls video downloading improved
- UI changes

<hr /><em>This discussion was created from the release <a href='https://github.com/Zenda-Cross/vega-app/releases/tag/v2.3.1'>v2.3.1</a>.</em>

**Comment by Mrunknow3** [comment]
Can you become a cs3 hindi provider and add dooflix bcz you already made dooflix you just need some changes for cs3 that's it.

**Reply by Zenda-Cross** [reply]
Yeah, but cs3 extensions are in Kotlin and I work in javascript

**Reply by Mrunknow3** [reply]
Ok no problem thanks for replying.

**Comment by Taiya30** [comment]
Can you add more anime extension's please 🥺 
and I really like your app thank youuu🤍

**Reply by Zenda-Cross** [reply]
Thank you so much
Trying my best here.
